compiler/modules/nback/number.bal

```diff
@@ -12,6 +12,15 @@ final RuntimeFunction floatToIntFunction = {
     attrs: ["nounwind", "readnone", "speculatable", "willreturn"]
 };
 
+final RuntimeFunction decimalNegateFunction = {
```

Comment:
This needs to be called decimalNegFunction (name is always same as runtime function name in camel case.)

Reply:
Changed